pkg/process/procutil/process_linux.go

```diff
@@ -76,7 +76,7 @@ func WithReturnZeroPermStats(enabled bool) Option {
 func WithPermission(enabled bool) Option {
```

Comment:
Could we also rename `WithPermission` to `WithElevatedPermissions`? I think it's much clearer that way

Reply:
Interestingly I had that change but then undid it realizing that name isn't congruent with the fact that accepts a boolean, if it were `WithElevatedPermissions()` it would be more consistent. I can rename `enabled bool` to `elevatedPermissions bool`?

Reply:
That will work!